thoth/messaging/admin_client.py

```diff
@@ -68,3 +68,17 @@ def create_topic(admin: AdminClient, message: MessageBase, partitions: int = 1,
             )
         ]
     )
+
+
+def check_connection(timeout: int = 10, config: Optional[Dict[str, str]] = None):
+    """Check connection to Kafka with either provided config or config gathered from env."""
+    if config:
+        a = AdminClient(config)
+    else:
+        a = AdminClient(kafka_config_from_env())
+
+    try:
+        a.list_topics(timeout=timeout)
+        return True
+    except Exception:
+        return False
```

Comment:
Would it make sense to report the exception here to get information in logs for further debugging?

Reply:
+1

Reply:
:+1: